01_01_Basic_Rules.txt

```diff
@@ -1,12 +1,20 @@
+==============================
+Basic Rules
+==============================
+	
+	Welcome to Compound X! The game of squad tactics, space exploration and 
+exciting role playing! On behalf of the develpment team, thanks for playing and
+have fun!
+
 ==============================
 Premise
 ==============================
 
-	Welcome to Compound X! The specific time and place that your adventure takes
-place is up to your Game Master (often called a GM for short), but all of them
-take place far in the future where humans have left earth on their faster-than-light
-spaceships to explore dangerous new worlds fraught with friend and foe of all
-manner of robots and aliens! 
+	The specific time and place that your adventure takes place is up to your 

```

Comment:
lang (sc)- I'd swap this to "time and place of your adventure is up to..." b/c it reads less awk

Reply:
s-v agreement: "adventures"